projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-column-docs/table-column-docs.component.ts

```diff
@@ -20,6 +20,7 @@ export class TableColumnDocsComponent {
   @Input() sortable?:  boolean = true;
   @Input() title:      string;
   @Input() width:      number;
+  @Input() wrapContent: boolean = null;
```

Comment:
Same comment as the other file:

Would it be better to give this a default value of false? I'm not sure of the implications of doing it null v. false here but I feel like if the user didn't define the input then it would at least have a false fall-back value.